components/Premium/PremiumFeatures/PremiumFeatures.tsx

```diff
@@ -22,62 +29,143 @@ function PremiumFeatures() {
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
                         <td className={styles.featureColumn}>Auction explorer</td>
-                        <td>{checkIcon}</td>
-                        <td>{checkIcon}</td>
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
                         <td className={styles.featureColumn}>Player auction history</td>
-                        <td>{checkIcon}</td>
-                        <td>{checkIcon}</td>
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
                         <td className={styles.featureColumn}>Display active auctions</td>
-                        <td>{checkIcon}</td>
-                        <td>{checkIcon}</td>
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
-                        <td className={styles.featureColumn}>Faster Flipper without delay</td>
-                        <td>{xIcon}</td>
-                        <td>{checkIcon}</td>
+                        <td className={styles.featureColumn}>Top 3 Kat flips</td>
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                    </tr>
+                    <tr>
+                        <td className={styles.featureColumn}>Top 3 Craft flips</td>
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
                         <td className={styles.featureColumn}>Full access to flipper filters</td>
-                        <td>{xIcon}</td>
-                        <td>{checkIcon}</td>
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                    </tr>
+                    <tr>
+                        <td className={styles.featureColumn}>
+                            List of low supply items (<Link href="/lowSupply">here</Link>)
+                        </td>
+                        {xIconElement}
+                        {checkIconElement}
```

Comment:
don't include this in starter for now
```suggestion
                        {xIconElement}
```